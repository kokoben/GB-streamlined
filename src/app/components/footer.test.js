import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './footer';

it('renders correctly', () => {
  const wrapper = shallow(<Footer />);
  const tree = toJson(wrapper);
  expect(tree).toMatchSnapshot();
});
