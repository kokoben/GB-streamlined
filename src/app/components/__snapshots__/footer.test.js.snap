// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Adapter
  style={
    Object {
      "color": "#fff",
      "marginBottom": "45px",
      "padding": "0",
      "textAlign": "center",
    }
  }
>
  Giant Bomb Streamlined Created by Ben Lee
</Adapter>
`;
